soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -133,33 +90,9 @@ struct ace_dfpmccu {
 
 #define ADSP_SHIM_DSPWCTCS_TTIE(c) BIT(8 + (c))
 
-/* Host memory window control.  Not strictly part of the shim block. */
-struct cavs_win {
-	uint32_t dmwba;
-	uint32_t dmwlo;
-};
-
-#define CAVS_WIN ((volatile struct cavs_win *)DT_REG_ADDR(DT_NODELABEL(win)))
-
```

Comment:
any reason to remove CAVS_WIN?  Not opposed to it, just curious.

Reply:
Just cleaned up every possible CAVS definition for the ACE platform. No other reason :)

Reply:
this is not being used anywhere, dts node win does not exist anymore.

Reply:
Thank you for the information. Good to know. 👍🏻